curriculum/structure/blocks/workshop-quincys-job-tips.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "Build a Quincy's Job Tips Page",
+  "isUpcomingChange": true,
```

This should be changed to false. we only set it to true for content that is not ready to be live on learn. But in this case, once this is merged in and deployed, campers can start working on it

```suggestion
  "isUpcomingChange": false,
```